### PR TITLE
10.x.x/hd/update wizard to handle quality asset

### PR DESCRIPTION
**Pull request by RSlysz**
### Purpose of this PR
Update of Wizard to better handle assets in Quality Settings
Fixed issue due to missing resource folder.
Fixed issue where DefaultVolumeProfile was checked against currentAsset instead of defaultAsset
Added LookDev Volume Profile check.
Remove the useless Populate/Reset (not really usefull since we don't have the custom scenes and duplicate with fixes under)
Updated documentation and added missing DXR update on documentation.
Unity_396kDlGKzs
Unity_5h8qJTsebJ

---
### Testing status
Manually tested in following context:
- Graphics: null ; Quality: null
- Graphics: HDRP ; Quality: null
- Graphics: URP ; Quality: null
- Graphics: null ; Quality: HDRP
- Graphics: HDRP ; Quality: HDRP
- Graphics: URP ; Quality: HDRP
- Graphics: null ; Quality: URP
- Graphics: HDRP ; Quality: URP
- Graphics: URP ; Quality: URP

That:
- introduced warning message above FixAll only display if the asset in Quality is of type URP
- introduced "Assigned - Quality" is red if asset in Quality of type URP
- both scope adapted to additional content in VR or DXR tab

---
### Comments to reviewers
Separate PR will also target:
- 2021.1/staging (v11): https://github.com/Unity-Technologies/Graphics/pull/3974
- master (v12): https://github.com/Unity-Technologies/Graphics/pull/4015

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md [hunk 1]
```diff
@@ -79,18 +93,31 @@ This tab provides all of the configuration options from the [HDRP tab](#HDRPTab)
...
-| **Screen Space Shadow**          | Checks to make sure **Screen Space Shadow** is enabled in the current [HDRP Asset](HDRP-Asset.md). <br />Press the **Fix** button to enable **Screen Space Shadow**. |
-| **Reflections** | Checks to make sure **Screen Space Reflection** is enabled in the current [HDRP Asset](HDRP-Asset.md). <br />Press the **Fix** button to enable **Screen Space Reflection**. |
-| **DXR Activated**                | Checks to make sure **DXR Activated** is enabled in the current [HDRP Asset](HDRP-Asset.md). <br />Press the **Fix** button to enable **DXR Activated**. |
-| **DXR Resources**               | Checks to make sure that your HDRP Asset references an **HD Render Pipeline RayTracing Resources**  Asset. <br />Press the **Fix** button to reload the raytracing resources for the HDRP Asset. |
+| **Architecture 64 bits** |  |
+| **DXR Resources** | Checks to make sure that your HDRP Asset references an **HD Render Pipeline RayTracing Resources**  Asset. <br />Press the **Fix** button to reload the raytracing resources for the HDRP Asset. |
+| **Screen Space Shadow (Default Camera Frame Settings)** |  |
+| **Reflections (Default Camera Frame Settings)** |  |
```
Comment (anisunity) [inline com.unity.render-pipelines.high-definition/Documentation~/Render-Pipeline-Wizard.md:106]: This is wrong

**Review by remi-chapelain — CHANGES_REQUESTED** [review]
Took a quick look and I suggest new text for what is highlighted in the image to be more coherent with the naming we have in HDRP. 

- Screen Space Shadow (Default Camera Frame Settings) => Screen Space Shadows (**HDRP Default Settings**)
- Reflection (Default Camera Frame Settings => **Screen Space Reflections (HDRP Default Settings)**
- Screen Space Reflection - Transparent (Default Camera Frame Settings => Screen Space Reflections - Transparent**s** (**HDRP Default Settings**)
- Global Illumination (Default Camera Frame Settings => **Screen Space Global Illumination (HDRP Default Settings)**
- For both SSR (HDRP Default Settings & Asset) I'd replace the circled red "**for**" with "**on**" to make it clearer that transparent will be able to receive reflections. 
- Also (in blue), the default settings for SSR - Transparents is not "On Transparent" but just "**Transparents**", I suggest renaming one or the other for coherence
image

- Finally, (underlined in black) is it worth mentioning the HDRP Asset name in the HDRP Default Settings item ? I know for 10.x.x Default settings are still embedder in the asset but should users needs to know where is it ?

image

**Review by JordanL8 — APPROVED** [review]

**Review by remi-chapelain — APPROVED** [review]
Aside from a little missing text that @RSlysz is adding, (for each Frame settings to inform users of the fact that to enable frame settings, you need to enable the feature in the asset first), we're good ! ✔️

**Review by sebastienlagarde — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs [hunk 2]
```diff
@@ -17,12 +18,6 @@ static class Style
 
             public const string hdrpProjectSettingsPathLabel = "Default Resources Folder";
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs:19]: note for future us: make sure those string can be localized

File: com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs [hunk 3]
```diff
@@ -53,6 +48,7 @@ static class Style
             public const string resolveAll = "Fix All";
             public const string resolveAllQuality = "Fix All Qualities";
             public const string resolveAllBuildTarget = "Fix All Platforms";
+            public const string fixAllOnNonHDRP = "Current Quality is not using High Definition render pipeline. If you attempt a Fix All, the current Quality will be changed to use it.";
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs:51]: I would rephrase with
The active Quality Level is not using a High Definition Render Pipeline asset. If you attempt a Fix All, the Quality Level will be changed to use it.

**Review by jenniferd-unity — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs (hunk 2 above)
Comment (RSlysz) [reply com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs:19]: Yes sorry it is the wold wizard. It should be a PR in itself as some label need to be extracted in a style class.
